src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -152,7 +152,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_LINKS: '2016-09-01',
         ResourceType.MGMT_RESOURCE_LOCKS: '2016-09-01',
         ResourceType.MGMT_RESOURCE_POLICY: '2019-09-01',
-        ResourceType.MGMT_RESOURCE_RESOURCES: '2020-10-01',
+        ResourceType.MGMT_RESOURCE_RESOURCES: '2020-06-01',
```

Comment:
Version bump caused our commands to fail with "APIVersion 2020-10-01 is not available" so I changed it back.. wondering if this will cause any newly added commands that needs the newest api version to fail?

Reply:
It is not an easy job to change API version. All tests that use this package will fail without rerecording them again.

Reply:
My mistake, reinstalled packages and working as expected